src/util/TypeTraits.h

```diff
@@ -215,4 +215,47 @@ template <typename... Ts>
 requires(sizeof...(Ts) > 0)
 using First = typename detail::FirstWrapper<Ts...>::type;
 
+/*
+@brief Require `Fn` to be invocable with `Args...` and the return type to be
+`isSimilar` with `Ret`.
```

Comment:
```suggestion
`isSimilar` to `Ret`.
```

Reply:
Also at the beginning of the block shortly describe the difference between these concepts and `std::invocable_r_v` or whatever they are called (where convertibility suffices).

Reply:
Done, see [80acf94](https://github.com/ad-freiburg/qlever/pull/1118/commits/80acf941d943fabb1d27aa684d4606eb2ac514b9).